packages/mobile/android/app/src/main/java/org/celo/mobile/MainActivity.java

```diff
@@ -56,6 +57,14 @@ public void onPause() {
       .setFlags(WindowManager.LayoutParams.FLAG_SECURE, WindowManager.LayoutParams.FLAG_SECURE);
   }
 
+  @Override
+  public void onNewIntent(Intent intent) {
+    Boolean firebaseEnabled = Boolean.parseBoolean(BuildConfig.FIREBASE_ENABLED);
+    if (firebaseEnabled) {
+      super.onNewIntent(intent);
+    }
+  }
+
```

Comment:
Why is this needed?

Reply:
So, with FIREBASE_DISABLED this causes a crash. The better fix would be to have a test firebase running in CI. I would love to do that but I am not sure how long it would take so in the interest of time for this milestone this was necessary.

Reply:
oh ok, is this a crash on the native side or JS?

Reply:
The crash is on the native side. It looks like the react activity tries to call firebase dynamic links, which in turn complains about firebase app not being initialized.

Reply:
Thanks, could you add a comment about this here?